src/rard/utils/convertors.py

```diff
@@ -31,6 +31,17 @@ def transfer_data_between_fragments(source, destination):
     destination.commentary = commentary
 
 
+def transfer_mentions(original, new):
+    if original.mentioned_in:
+        for tof in original.mentioned_in.all():
+            tof.reassign_mentions(original, new)
+            tof.update_content_mentions()
+            tof.update_mentions()
```

Comment:
Can you add a comment next to each function call to summarise what it's doing?
e.g. reassigning mentions in the tof content, updating mention display text, updating database relations...